packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -218,49 +218,53 @@ export const ClayPaginationBarWithBasicItems = ({
 	}, [totalPages]);
 
 	return (
```

Comment:
I don't like this approach of not rendering the element internally it can cause a lot of confusion, I think what we can do here is if the `totalItems` is `0` we change it to `1` which would be the minimum acceptable.

This change can be done in Pagination as well because it will have the same error if only it is used.